File: app/src/main/java/com/akexorcist/lovelyrecyclerview/adapter/holder/TitleViewHolder.java
package com.akexorcist.lovelyrecyclerview.adapter.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.akexorcist.lovelyrecyclerview.R;


/**
 * Created by Akexorcist on 10/28/2016 AD.
 */

public class TitleViewHolder extends RecyclerView.ViewHolder {
    public TextView tvTitle;

    public TitleViewHolder(View itemView) {
        super(itemView);
        tvTitle = (TextView) itemView.findViewById(R.id.tv_title);
    }
}

File: app/src/main/java/com/akexorcist/lovelyrecyclerview/adapter/holder/TitleViewHolder.kt
package com.akexorcist.lovelyrecyclerview.adapter.holder

import androidx.recyclerview.widget.RecyclerView
import com.akexorcist.lovelyrecyclerview.adapter.model.OrderDetailItem
import com.akexorcist.lovelyrecyclerview.databinding.ViewTitleBinding

class TitleViewHolder(private val binding: ViewTitleBinding) : RecyclerView.ViewHolder(binding.root) {
    fun bind(item: OrderDetailItem.Title) {
        binding.tvTitle.text = item.title
    }
}
